pages/index.page.tsx

```diff
@@ -212,26 +212,23 @@ const Home = (props: any) => {
             <div className='lg:w-[650px]  mx-auto my-10 grid grid-cols-1 lg:grid-cols-3 gap-8 justify-items-center '>
               <Link
                 href='/learn/getting-started-step-by-step'
-                className=' flex items-center justify-center rounded border-2 border-white dark:border-none text-white w-[194px] h-[40px] font-semibold bg-primary dark:shadow-2xl'
+                className='flex items-center justify-center rounded border-2 border-white hover:bg-blue-700 transition-all duration-300 ease-in-out text-white w-[194px] h-[40px] font-semibold'
               >
                 Getting started
               </Link>
               <Link
                 href='/slack'
-                className=' flex items-center justify-center rounded border-2 border-white dark:border-none text-white  w-[194px] h-[40px] font-semibold bg-primary dark:border-shadow-white  dark:shadow-2xl'
+                className='flex items-center justify-center rounded border-2 border-white hover:bg-blue-700 transition-all duration-300 ease-in-out text-white  w-[194px] h-[40px] font-semibold'
               >
                 Join Slack
               </Link>
 
-              <div className='flex herobtn items-center justify-center font-semibold w-[194px] h-[40px] rounded border-2 border-white dark:border-none text-white bg-primary mx-auto  dark:shadow-2xl'>
-                <div className='flex flex-row justify-center items-center mr-4 '>
-                  <DocSearch
-                    appId='6ZT4KX2OUI'
-                    apiKey='69f76fba13585144f6686622e9c8f2a8'
-                    indexName='json-schema'
-                  />
-                  <p>Search</p>
-                </div>
+              <div className='herobtn rounded border-2 border-white hover:bg-blue-700 transition-all duration-300 ease-in-out text-white mx-auto'>
```

Comment:
@VivekJaiswal18 pls. have a look on this change. because after applying this the search bar UI is not as expected.

![image](https://github.com/json-schema-org/website/assets/124715224/6a320cf1-6aa6-4a05-86fe-c879ea03dab4)